hermes-console-vue/src/api/subscription-metrics.ts

```diff
@@ -0,0 +1,17 @@
+export interface SubscriptionMetrics {
+  delivered: number;
+  discarded: number;
+  volume: number;
+  timeouts: MetricDecimalValue;
```

Comment:
Can't we just use string here? I think creating type for this value doesn't give us anything. I think providing type would be beneficial in case of some kind of api that provides some set of possible values